Command/GenKeysCommand.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Lexik\Bundle\JWTAuthenticationBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * GenKeysCommand.
+ *
+ * @author zorn-v
+ */
+class GenKeysCommand extends Command
+{
+    /**
+     * @var string
+     */
+    private $privateKey;
+
+    /**
+     * @var string
+     */
+    private $publicKey;
+
+    /**
+     * @var string
+     */
+    private $passphrase;
+
+    /**
+     * Constructor.
+     *
+     * @param string $privateKey
+     * @param string $publicKey
+     * @param string $passphrase
+     */
+    public function __construct($privateKey, $publicKey, $passphrase)
+    {
+        $this->privateKey = $privateKey;
+        $this->publicKey  = $publicKey;
+        $this->passphrase = $passphrase;
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('lexik:jwt:gen-keys')
+            ->setDescription('Generate JWT keys')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $fs = new Filesystem();
+        $fs->mkdir(dirname($this->privateKey));
+        $fs->mkdir(dirname($this->publicKey));
+
+        $pKeyRes = openssl_pkey_new([
+            'digest_alg'         => 'sha512',
+            'private_key_bits'   => 4096,
+            'private_key_type'   => OPENSSL_KEYTYPE_RSA,
```

Comment:
This bundle also supports ECDSA, the command should probably throw an exception in this case (and maybe support it in the future).